README.md

```diff
@@ -421,9 +421,13 @@ between Gunicorn runs (before startup is recommended).
 
 Put the following in the config file:
 ```python
-def worker_exit(server, worker):
-    from prometheus_client import multiprocess
-    multiprocess.mark_process_dead(worker.pid)
+def child_exit(server, worker):
+    import logging
```

Comment:
So you are suggest that also `from prometheus_client import multiprocess` has to be moved?

I will need to re-test then, if the try catch is still needed after extracting the import.